docs/topics/catalyst/catalyst.md

```diff
@@ -0,0 +1,142 @@
+# Gems & Catalyst
+
+## Introduction
+
+The aim of this document is to explain how Gems & catalysts work and how they are tracked by the smart contracts.
+
+Roughly a **catalyst** is a container for **gems**. The rarer the catalyst, the more gems it may contain. The user can then equip his asset with a catalyst in order to add ability to it. The gems are of different **types** (luck, power...). Depending on their types, the gems will have a different effect in games. One Gem will be able to provide up to 25 **attribute** points of a specific type. 
+
+From now on we will call gems and catalysts G&C.
+
+G&C in action:
+![](https://miro.medium.com/max/3600/0*9tVheYzwmmALkJBa)
+
+For more information see: <https://medium.com/sandbox-game/presenting-the-sandbox-gems-catalysts-f017a18ff5fb>
+
+### Catalyst data
+
+Retrieve from [here](https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/data/catalysts.ts)
+
+| Id  | Catalyst type | MaxGems |
+| --- | ------------- | ------- |
+| 1   | COMMON        | 1       |
+| 2   | RARE          | 2       |
+| 3   | EPIC          | 3       |
+| 4   | LEGENDARY     | 4       |
+
+### Gem data
+Retrieve from [here](https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/data/gems.ts)
+
+| Id  | Gem type | Attribute point |
+| --- | -------- | --------------- |
+| 1   | POWER    | 1..25           |
+| 2   | DEFENSE  | 1..25           |
+| 3   | MAGIC    | 1..25           |
+| 4   | LUCK     | 1..25           |
+
+More on attribute point logic [here](https://sandboxgame.gitbook.io/the-sandbox/assets/gems-and-catalysts/attributes-and-behaviours)
+
```

Comment:
Can we move reorganize the content like this

```
## Introduction
## Model
### Catalyst (you can put your catalyst data, and link to the contract in here)
### Gem
### Registry
### Class diagram
## Processes
(...)
```

Also, can we add a table with the properties of each smart contracts / token in the intro of entity (catalyst, gem, registry) ? Something like this (taken from land doc)

```
|              Feature | Link                                   |
| --------------------:|:-------------------------------------- |
|            Contract | [PolygonLand](http://) |
|            `ERC-165` | https://eips.ethereum.org/EIPS/eip-165 |
|            `ERC-721` | https://eips.ethereum.org/EIPS/eip-721 |
|   `ERC-721 metadata` | Il manque l'interface IERC721Metadata        |
|            `ERC2771` |              (...)                          |
|        `Upgradeable` | No                                     |
|             `Minter` |                   (...)                     |
| `WithSuperOperators` |           (...)                             |
|                      |                                        |
```